docs/guides/auth.md

```diff
@@ -192,6 +192,43 @@ channel = implementations.secure_channel('localhost', 50051, creds)
 stub = helloworld_pb2.beta_create_Greeter_stub(channel)
 ```
 
+###SSL/TLS for server authentication and encryption (Java)
+
+In Java we recommend that you use OpenSSL when using gRPC over TLS. You can find details about installing and using OpenSSL and other required libraries for both Android and non-Android Java in the gRPC Java [Security](https://github.com/grpc/grpc-java/blob/master/SECURITY.md#transport-security-tls) documentation.
+
+To enable TLS on a server, a certificate chain and private key need to be specified in PEM format. The standard TLS port is 443, but we use 8443 below to avoid needing extra permissions from the OS.
+
+```
+ServerImpl server = ServerBuilder.forPort(8443)
```

Comment:
Oops. `ServerImpl` should be just `Server`. That was changed a bit ago. I'll fix up `SECURITY.md`.